docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -248,7 +255,34 @@ complete this procedure.
 
 .. _start-rest-api-label:
```

Comment:
look like you need to remove the label here since you redefine it below. It should probably be changed to a label for Start the DevMode Consensus Engine

Reply:
Fixed. I renamed it `_start-devmode-consensus-label:`
Thanks for looking at this.